core/domain/feedback_services.py

```diff
@@ -291,12 +308,39 @@ def _add_feedback_message_reference(user_id, reference):
         enqueue_feedback_message_email_task(user_id)
 
 
-def send_feedback_message_email(exploration_id, thread_id, message_id):
+def update_feedback_email_retries(user_id):
+    model = feedback_models.UnsentFeedbackEmailModel.get(user_id)
+    time = (datetime.datetime.utcnow() - model.created_on).seconds
```

Comment:
I did some [googling](https://cloud.google.com/appengine/docs/python/datastore/typesandpropertyclasses#Property_Classes) and found that date and time property stores utc time by default. And also when I did backend manual testing, I set the countdown to 20 seconds to note the time of model execution and it did execute handler function after 20 seconds, so I guess we do not have to worry about it.